### PR TITLE
support force up in juniper switches

### PR DESCRIPTION
**Pull request by vilnitsky**

### CODE REVIEW

File: tests/juniper/juniper_base_protocol_test.py [hunk 1]
```diff
@@ -1669,6 +1669,21 @@ def test_that_an_interface_cannot_be_deleted_while_there_is_still_an_rstp_config
...
+                {"interface": [
+                    {"name": "ge-0/0/1"},
+                    {"ether-options": {
+                        "ieee-802.3ad": {"lacp": {"force-up": ""}},
+                    }}]},
+            ]})
+        self.nc.commit()
+        get_interface_reply = self.nc.get_config(source="running", filter=dict_2_etree({"filter": {
+            "configuration": {"interfaces": {"interface": {"name": "ge-0/0/1"}}}}}))
+        assert_that(get_interface_reply.xpath("data/configuration"), has_length(1))
```
Comment (fbouliane) [inline tests/juniper/juniper_base_protocol_test.py:1684]: something like that would asses more what you're looking for. 

        assert_that(get_interface_reply.xpath("data/configuration/interfaces/interface/ether-options/*")[0].tag, is_('ieee-802.3ad'))
        assert_that(get_interface_reply.xpath("data/configuration/interfaces/interface/ether-options/ieee-802.3ad/*")[0].tag, is_('lacp'))
        assert_that(get_interface_reply.xpath("data/configuration/interfaces/interface/ether-options/ieee-802.3ad/lacp/*")[0].tag, is_('force-up'))

File: tests/juniper/juniper_base_protocol_test.py [hunk 2]
```diff
@@ -1669,6 +1669,21 @@ def test_that_an_interface_cannot_be_deleted_while_there_is_still_an_rstp_config
 
         self.cleanup(reset_interface("ge-0/0/3"))
 
+    def test_that_force_up_can_be_configured(self):
```
Comment (fbouliane) [inline tests/juniper/juniper_base_protocol_test.py:1672]: pretty sure you need a way to remove the force-up as well
Does it goes away when you delete the interface ? 
do we have a way of unsetting this specifically ? 

I would like to see a test where you remove it the same way we do on the switch.

**Review by fbouliane — COMMENTED** [review]
this should be ok,
i'd like the removal to be added,
we are working on bringin back unit test.

File: tests/juniper/juniper_base_protocol_test.py (hunk 2 above)
Comment (fbouliane) [reply tests/juniper/juniper_base_protocol_test.py:1672]: looks like  this according to your netman PR 

        <ieee-802.3ad>
            <lacp operation=\"delete\"></lacp>
        </ieee-802.3ad>

Comment (vilnitsky) [reply tests/juniper/juniper_base_protocol_test.py:1672]: yes, removing force-up was missed. I have added and its tests

**Review by fbouliane — APPROVED** [review]

**Pull request merged**